modules/develop/pages/data-transforms/how-transforms-work.adoc

```diff
@@ -50,15 +50,21 @@ If the previous instance of the transform function failed to commit its latest o
 
 This section outlines the limitations of data transforms. These constraints are categorized into general limitations affecting the overall functionality and specific limitations related to giving data transforms access to custom environment variables.
 
-=== General limitations
+=== General
 
 - *No external access*: Transform functions have no external access to disk or network resources.
 - *Single record transforms*: Only single record transforms are supported, but multiple output records from a single input record are supported. For aggregations, joins, or complex transformations, consider xref:redpanda-connect:ROOT:about.adoc[Redpanda Connect] or https://flink.apache.org/[Apache Flink].
 - *Output topic limit*: Up to eight output topics are supported.
 - *Delivery semantics*: Transform functions have at-least-once delivery.
 - *Transactions API*: When clients use the Kafka Transactions API on partitions of an input topic, transform functions process only committed records.
 
-=== Environment variable limitations
+=== JavaScript
+
+- *No native extensions*: Native Node.js extensions are not supported. You cannot use packages that require compiling native code or interacting with low-level system features.
+- *Limited Node.js standard modules*: You can use only modules that can be polyfilled by the https://www.npmjs.com/package/esbuild-plugin-polyfill-node#implemented-polyfills[esbuild plugin]. Even if a module can be polyfilled, certain functionalities, such as network connections, will not work because the necessary browser APIs are not exposed in the Redpanda JavaScript runtime environment. For example, while the plugin can provide stubs for some Node.js modules such as `http` and `process`, these stubs will not work in the Redpanda JavaScript runtime environment.
```

Comment:
```suggestion
- *Limited Node.js standard modules*: Only modules that can be polyfilled by the https://www.npmjs.com/package/esbuild-plugin-polyfill-node#implemented-polyfills[esbuild plugin^] can be used. Even if a module can be polyfilled, certain functionalities, such as network connections, will not work because the necessary browser APIs are not exposed in the Redpanda JavaScript runtime environment. For example, while the plugin can provide stubs for some Node.js modules such as `http` and `process`, these stubs will not work in the Redpanda JavaScript runtime environment.
```